internal/state/store.go

```diff
@@ -408,60 +420,65 @@ func (store dbStore) reverseBatchDelete(batch dbm.Batch, start, end []byte) ([]b
 
 //------------------------------------------------------------------------
 
-// LoadABCIResponses loads the ABCIResponses for the given height from the
-// database. If not found, ErrNoABCIResponsesForHeight is returned.
+// LoadFinalizeResponses loads the responses to FinalizeBlock for the given height from the
+// database. If not found, ErrNoFinalizeResponsesForHeight is returned.
 //
 // This is useful for recovering from crashes where we called app.Commit and
 // before we called s.Save(). It can also be used to produce Merkle proofs of
 // the result of txs.
-func (store dbStore) LoadABCIResponses(height int64) (*tmstate.ABCIResponses, error) {
-	buf, err := store.db.Get(abciResponsesKey(height))
+func (store dbStore) LoadFinalizeResponses(height int64) (*abci.ResponseFinalizeBlock, error) {
```

Comment:
We can. I considered it for a while and thought it would be too long.
But since you raise the point, I guess it'll be clarity over length :-)
Will change it